lab_2_keywords_cooccurrence/main.py

```diff
@@ -3,11 +3,32 @@
 Extract keywords based on co-occurrence frequency
 """
 from pathlib import Path
-from typing import Optional, Sequence, Mapping
+from typing import Optional, Sequence, Mapping, Any
+
 
 KeyPhrase = tuple[str, ...]
 KeyPhrases = Sequence[KeyPhrase]
 
+def check_str(user_input: Any) -> bool:
+    """
+    Checks if the input is a string and it is not empty
+    """
+    if not user_input or not isinstance(user_input, str):
```

Comment:
You can return the result of the condition without `not`s and it should be prettier

Reply:
Done